app/models/analyticEvent.server.ts

```diff
@@ -36,21 +56,80 @@ export async function getAnalyticsDataLast7Days(): Promise<Array<ARes>> {
   const analyticsData = await prisma.analyticEvent.groupBy({
     where: {
       ts: {
-        gte: startDate.toISOString(),
-      },
+        gte: startDate.toISOString()
+      }
     },
-    by: ['event', 'data', 'ip'],
+    by: ["event", "data", "ip"]
   });
 
-  const map = new Map<string,ARes>();
-  analyticsData.forEach(x => {
-    const key = x.event+x.data;
-    const prev = map.get(key) ?? {event:x.event, data:x.data, count:0};
-    prev.count +=1;
+  const map = new Map<string, ARes>();
+  analyticsData.forEach((x) => {
+    const key = x.event + x.data;
+    const prev = map.get(key) ?? { event: x.event, data: x.data, count: 0 };
+    prev.count += 1;
     map.set(key, prev);
   });
 
   var res = [...map.values()];
   res = res.sort((a, b) => b.count - a.count);
   return res;
-}
\ No newline at end of file
+}
+
+export async function tallyAnalytics(): Promise<Array<ARes>> {
+  const startDate = new Date();
+  startDate.setDate(startDate.getDate() - 2);
+
+  const analyticsData = await prisma.analyticEvent.findMany({
+    where: {
+      ts: {
+        lt: startDate.toISOString()
+      }
+    }
+  });
+
+  const map = new Map<string, ARes>();
```

Comment:
i don't see this keeping track of separate counts for each day?
this looks like it will just store the total tally of all events ever?

Reply:
Resolved.